### PR TITLE
Bug Fixes

### PR DESCRIPTION
**Pull request by pancho111203**
- Build fix that made 'async' not work
- Bug that forced you to use the name of the query instead of the query field name:

 ```js
      query getAccountants($perPage: Int) {
        getAccountantsAdmin(limit: $perPage) {
          items {
            id,
            name
          },
          totalCount
          __typename
        }
      }
```

This threw an error, and forced you to type it like this: 

 ```js
      query getAccountantsAdmin($perPage: Int) {
        getAccountantsAdmin(limit: $perPage) {
          items {
            id,
            name
          },
          totalCount
          __typename
        }
      }
```

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -3,12 +3,25 @@
...
+          "istanbul",
+          {
+            "exclude": "**/*.spec.js"
+          }
+        ]
       ]
     }
   }
-}
+}
```
Comment (djhi) [inline .babelrc]: Please don't modify formatting and add empty end line

File: package.json [hunk 2]
```diff
@@ -29,7 +29,8 @@
     "nyc": "10.1.2"
   },
   "dependencies": {
-    "apollo-client": "0.8.5",
+    "apollo-client": "^0.8.7",
```
Comment (djhi) [inline package.json]: If you have to change the dependencies, please use `~` and while you're at it, can you add the `~` to the others as well ?

File: src/parseApolloResponse.js [hunk 3]
```diff
@@ -13,7 +13,7 @@ import {
  */
 const getApolloResultKey = (type, apolloParams) => (
     QUERY_TYPES.includes(type)
-    ? apolloParams.query.definitions[0].name.value
+    ? apolloParams.query.definitions[0].selectionSet.selections[0].name.value
```
Comment (djhi) [inline src/parseApolloResponse.js]: :+1:

**Comment by pancho111203** [comment]
@djhi The tests are using also the query name, that's why they are failing:

```js
const apolloParams = {
            query: {
                definitions: [{ name: { value: 'operationName' } }],
            },
        };
```

I will change them, I hope that's alright!

**Comment by djhi** [comment]
Published in v0.0.2. Thanks !

**Pull request merged**